dev-haskell/postgresql-typed/postgresql-typed-0.6.2.5.ebuild

```diff
@@ -34,22 +32,36 @@ RDEPEND=">=dev-haskell/attoparsec-0.12:=[profile?] <dev-haskell/attoparsec-0.15:
 			text? ( >=dev-haskell/text-1:=[profile?] )
 			uuid? ( >=dev-haskell/uuid-1.3:=[profile?] ) )
 	hdbc? ( >=dev-haskell/hdbc-2.2:=[profile?] )
-	md5? ( >=dev-haskell/cryptonite-0.5:=[profile?]
-		>=dev-haskell/memory-0.5:=[profile?] )
+	md5? ( >=dev-haskell/memory-0.5:=[profile?]
+		crypton? ( dev-haskell/crypton:=[profile?] )
+		!crypton? ( >=dev-haskell/cryptonite-0.5:=[profile?] )
+		tls? ( crypton? ( dev-haskell/crypton-x509:=[profile?]
+					dev-haskell/crypton-x509-store:=[profile?]
+					dev-haskell/crypton-x509-validation:=[profile?]
+					>=dev-haskell/tls-1.7:=[profile?] )
+			!crypton? ( <dev-haskell/tls-1.7:=[profile?]
```

Comment:
We need to hard-enable `crypton` as the only option in hackport flags. We can't allow downgrading to such an old version of tls, as it would not work with the rest of the ecosystem.